docs/freqtrade-101.md

```diff
@@ -0,0 +1,188 @@
+# Freqtrade Strategies 101: A Quick Start for Strategy Development
+
+For the purposes of this quick start, we are assuming you are familiar with the basics of trading, and have read the 
+[Freqtrade basics](bot-basics.md) page.
+
+## Required Knowledge
+
+A strategy in Freqtrade is a Python class that defines the logic for buying and selling cryptocurrency `assets`.
+
+Assets are defined as `pairs`, which represent the `coin` and the `stake`. The coin is the asset you are trading using another currency as the stake.
+
+Data is supplied by the exchange in the form of `candles`, which are made up of a six values: `date`, `open`, `high`, `low`, `close` and `volume`.
+
+`Technical analysis` functions analyse the candle data using various computational and statistical formulae, and produce secondary values called `indicators`.
+
+Indicators are analysed on the asset pair candles to generate `signals`.
+
+Signals are turned into `orders` on a cryptocurrency `exchange`, i.e. `trades`.
+
+We use the terms `entry` and `exit` instead of `buying` and `selling` because Freqtrade supports both `long` and `short` trades.
+
+- **long**: You buy the coin based on a stake, e.g. buying the coin BTC using USDT as your stake, and you make a profit by selling the coin at a higher rate than you paid for. Profits are made in long trades by the coin value going up versus the stake.
+- **short**: You borrow capital from the exchange in the form of the coin, and you pay back the stake value of the coin later. Profits are made in short trades by the coin value going down versus the stake (you pay the loan off at a lower rate).
```

Comment:
`Profits are made in long trades by the coin value going up versus the stake.`
the 2nd sentence each sounds a bit odd ... can't tell exactly why - maybe "in long trades, profits are made..." would sound better?
I'm also not sure if "going up versus the stake" couldn't be rephrased into "by the price of the coin raising" ?